projects/components/src/popover/popover.component.ts

```diff
@@ -62,6 +69,30 @@ export class PopoverComponent implements OnDestroy {
 
   @HostListener('click')
   public onClick(): void {
+    if (this.popoverTriggerType === PopoverTriggerType.Click) {
+      this.handleClickPopoverTrigger();
+    }
+  }
+
+  @HostListener('mouseenter')
+  public onMouseenter(): void {
+    if (this.popoverTriggerType === PopoverTriggerType.Hover) {
+      this.handleHoverPopoverTrigger('mouseenter');
```

Comment:
Please don't hardcode a method param to get completely separate behavior. Suggest just making two methods - handlePopoverOpen, handlePopoverClose. Ideally, should make the click method delegate as needed.